### DISCUSSION TITLE
Cookbook App can't connect to local nc

### DISCUSSION THREAD

**Discussion by playsier** [nextcloud/cookbook#2603] [FAQ and user questions]
Hey there,

thank you guys for taking time and read this small questionary.
I currently run NC on 29.0.9. I recently switched from an "online" cloud (accessible via website) to a local NC (only private use).

I also wanted to change the cookbook path inside of the iOS App because the app itself told me, it lost its connection (ofc, I changed paths). So I went on and resetted the Coobook App. Turns out, I have to log in again (makes sense). When entering the local adress of the NC (e.g. nc.local) on the "Nextcloud Login" tab it says "Unable to reach server. Please check your server adress and internet connection.". My question is wether it is even possible, to use the Cookbook App with a local NC. The NC-app itself works flawlessly. Also tried the local IP-adress of the cloud, resulting in the same message.

Can you guys give me any tips?
Thanks a lot!

Greetings!

**Comment by christianlupus** [comment]
Can a browser on the phone resolve the URL name?
In general, this is the wrong audience here: We are only working on the app on the NC instance. The iOS app is not associated with this repository, so, I can only give basic help here.

**Reply by playsier** [reply]
Yes, for instance, when I open the local link (doesn't matter wether IP-adress or nc.local-link) it opens the clouds' dashboard without any problem. But trying to open it in the apps login page doesn't.

Is the iOS App repository nearby or should I check the AppStore for further information on the team?

**Reply by christianlupus** [reply]
Most probably, you can find a link here: https://github.com/nextcloud/cookbook/blob/master/docs/user/clients/Index.md
